requirements/dev.txt

```diff
@@ -1,97 +1,217 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.6
```

Comment:
When we can afford later on, I think it's worth continuing @lmarinve discussion regarding deps mgmt. development dependencies are being freeze a bit too strictly when generating with `pip-compile`, I think when we need to start supporting other Python version like we already have on `kytos` core, we might start hitting issues such as conflicting deps or unsupported versions when upgrading, because either the dep isn't ready to move on, delegating this to be resolved more conveniently and deterministically to a package manager would be great. cc'ing @italovalcy @ajoaoff @lmarinve

Reply:
I've created an issue for this discussion: https://github.com/kytos-ng/flow_manager/issues/40